remote-config.jsonc

```
@@ -19,10 +19,10 @@
     "notifications": {
       "interval": 20,
       "infos": [
-        //        {
-        //          "message": "The upstream deb.sury.org package key was unfortunately allowed to come near expiration as of 16 Feb 2024. This does not affect DDEV v1.22.7, so upgrade, but if you see warnings or errors in earlier versions during `ddev start` the easy fix is in https://github.com/ddev/ddev/issues/5795",
-        //          "versions": "<=v1.22.6"
-        //        }
+        {
+          "message": "Please upgrade to latest DDEV v1.24.3+ to avoid potential problems with new upcoming versions of Docker providers. https://github.com/ddev/ddev/releases",
```

```suggestion
          "message": "Please upgrade to latest DDEV v1.24.3+ to avoid potential problems with newer versions of Docker providers. https://github.com/ddev/ddev/releases",
```

I don't think we need "new" and "upcoming". Because the version is already out, I went with "new"

---

Oh, I see. You mentioned on [Slack](https://drupal.slack.com/archives/C5TQRQZRR/p1740583078269589?thread_ts=1740579744.794869&cid=C5TQRQZRR) that some docker-providers like Orbstack haven't updated to newest docker versions.

---

```suggestion
          "message": "Please upgrade to latest DDEV v1.24.3+ to avoid potential problems with upcoming versions of Docker providers as they start to support Docker 28. https://github.com/ddev/ddev/releases",
```